README.md

```diff
@@ -3,6 +3,9 @@ A place to organize style guides, best practices, tools, and techniques for Stan
 
  - [Best Practices](/best-practices)
    - [Documentation](/best-practices/documentation)
+   - [How To - Readme](/best-practices/howto_readme.md)
+   - [Licensing](/best-practices/licensing.md)
+   - [Version Control](/best-practices/version_control.md)
```

Comment:
Perhaps we could experiment with a utility to generate a table of contents, e.g.
  - http://brettterpstra.com/2014/07/01/github-toc-service/

For ease of use, if it works, it might be enabled with a git commit hook.